### ISSUE TITLE
transition: Routing calculation: Use distances instead of times to access/egress

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#190]
Some transportation societies have service levels calculated in distances instead of time, so the transit routing form and accessibility map form should add the possibility to enter distances instead of time and calculations should be updated accordingly.
